src/benchmark/md5.c

```diff
@@ -0,0 +1,10 @@
+#include <openssl/md5.h>
```

Comment:
The present structure makes it pretty easy to add other functions. I'll add in SHA1/SHA256 comparisons too.